specification/azurefleet/AzureFleet.Management/examples/2023-11-01-preview/Fleets_CreateOrUpdate_MaximumSet.json

```diff
@@ -765,7 +765,8 @@
     },
     "201": {
       "headers": {
-        "Azure-AsyncOperation": "https://contoso.com/operationstatus"
+        "Azure-AsyncOperation": "https://contoso.com/operationstatus",
+        "Retry-After": 30
```

Comment:
I don't see Retry-after being returned in response. Also retry after is optional field now from ARM. We use `x-ms-ratelimit-remaining-resource` for throttling. 
![image](https://github.com/user-attachments/assets/dfe58291-53ff-4f3d-abde-6cb451700d00)

Reply:
Removed from Examples.

Reply:
Removed it from examples.